code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -114,3 +126,5 @@
 
 		if(empty)
 			. += "<span class='notice'>It is currently empty. Allow some time for the internal syntheszier to produce more.</span>"
+
+#undef BORGHYPO_REFILL_VALUE
```

Comment:
Forgot a line ending after this one. The last line always should be an empty line.

Reply:
done, thank you